test/server/cards/01_SOR/units/BazeMalbusTempleGuardian.spec.ts

```diff
@@ -0,0 +1,80 @@
+                        },
+                        player2: {
+                            groundArena: ['wampa', 'battlefield-marine'],
+                            hasInitiative: true
+                        }
+                    });
+                });
+
+                // Checks both that sentinel is not active initially, and then activates after player claims initiative.
+                it('should not have sentinel until the player claims initiative', function () {
+                    const { context } = contextRef;
+
+                    expect(context.player2).toBeActivePlayer();
+                    context.player2.clickCard(context.wampa);
+                    context.player2.clickCard(context.p1Base);
+                    expect(context.p1Base.damage).toBe(4);
+                    expect(context.player1).toHaveClaimInitiativeAbilityButton();
+                    context.player1.claimInitiative();
+                    
+                    expect(context.player2).toBeActivePlayer();
+                    context.player2.clickCard(context.battlefieldMarine);
+                    // Baze Malbus automatically selected due to sentinel
+                    expect(context.battlefieldMarine.damage).toBe(2);
+                    expect(context.player1).toBeActivePlayer();
+                    expect(context.bazeMalbus.damage).toBe(3);
+                });
+            });
+
+            describe('when the player has initiative', function () {
+                beforeEach(function () {
+                    contextRef.setupTest({
+                        phase: 'action',
+                        player1: {
+                            groundArena: ['baze-malbus#temple-guardian'],
+                            hasInitiative: true
+                        },
+                        player2: {
+                            groundArena: ['wampa', 'battlefield-marine'],
+                        }
+                    });
+                });
+
+                // Checks that sentinel works initially, and then after it is claimed sentinel is no longer active.
+                it('has sentinel until initiative is claimed by opposing player', function () {
+                    const { context } = contextRef;
+
+                    expect(context.player1).toBeActivePlayer();
+                    context.player1.passAction();
+
+                    context.player2.clickCard(context.battlefieldMarine);     
+                                   
+                    // Baze Malbus automatically selected due to sentinel
+                    expect(context.battlefieldMarine.damage).toBe(2);
+                    expect(context.bazeMalbus.damage).toBe(3);
+
+                    context.player1.clickCard(context.bazeMalbus);
+                    context.player1.clickCard(context.p2Base);
+                    
+                    expect(context.player2).toHaveClaimInitiativeAbilityButton();
```

Comment:
Just for readability it would be helpful to add comment descriptors by case, since we have to make these so long - see GeneralKrellHeartlessTactician.spec.js for an example (doesn't have to look exactly like that, but just convey similar information would be great for folks trying to read the test for the first time in case someone breaks it)

Reply:
Added comments for each case within a test.